_posts/2021-10-12-devfile-v2-and-the-devworkspace-operator-p1.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Devfile v2 and the DevWorkspace Operator - Part 1
+layout: post
+author: Mario Loriedo
+description: >-
+  How to enable the DevWorkspace engine and what it means from a user point of view
+categories: []
+keywords: ['devfile', 'devworkpace']
+slug: /@mario.loriedo/devfile-v2-and-the-devworkspace-operator-p1
+---
+
+![Locally trusted certs]({{ site.url }}{{ site.baseurl }}/assets/img/devfile-v2-and-the-devworkspace-operator-part1/che-workspace-engines.png)
+
+With version [7.28 of Che](https://github.com/eclipse/che/releases/tag/7.28.0) we have introduced a workspace engine, [the DevWorkspace Operator](https://github.com/devfile/devworkspace-operator), that supports v2 of the Devfile specification. Although the default engine is still the che-server we plan to replace it with the DevWorkspace in the next few months.
+
+Switching to the DevWorkspace engine has some important consequences. Notably on the authentication subsystem that will be lighter and more flexible, on the workspaces network managed by a central gateway powered by [Traefik](https://github.com/traefik/traefik) and simpler configuration options for Che administrators.  
+
+This is the first of a series of three articles reviewing the changes introduced with the DevWorkspace. Here we will discuss the changes from the point of view of a Che user. The second part will be about the point of view of a Che administrator. The last part will be dedicated to the DevWorkspace Operator.
+
+## How to enable the DevWorkspace Operator 
+Che default workspace engine is the che-server. The DevWorkspace engine needs to be explicitly enabled. The following [chectl](https://github.com/che-incubator/chectl) command deploys Che on OpenShift configured with the DevWorkspace Operator as workspace engine:
+
+```bash
+chectl server:deploy --installer=olm -p openshift \
```

Comment:
```suggestion
chectl server:deploy -p openshift \
```
--> Installer type. If not set, default is "olm" for OpenShift 4.x platform otherwise "operator".